connectors-home/modules/ROOT/partials/oauth2-description.adoc

```diff
@@ -0,0 +1,3 @@
+// Use this partial in the Reference and Studio page (if the connector has one) as the description for OAuth2 in the respective OAuth2 section, so it references the two general OAuth2 configuration topics.
+
```

Comment:
This reads nicely. I would flip the last sentence around to say "For more information, refer to..." or "For more information, see..." The reason for doing this is that when the "For more information" is at the end of the sentence, users see the links before knowing what they are for. This is very subtle, but it follows the principle of technical writing of not having users read extra words. After reading "For more information," they can decide whether to continue reading.